db/migrate/20080625185118_create_users.rb

```diff
@@ -1,7 +1,8 @@
 class CreateUsers < ActiveRecord::Migration
   def self.up
     create_table :users do |t|
-      t.string     :username, :realname, :email, :hashed_password, :location, :gamertag, :avatar_url
+      t.string     :username, limit: 100
+      t.string     :realname, :email, :hashed_password, :location, :gamertag, :avatar_url
```

Comment:
Line is too long. [89/80]<br>Unnecessary spacing detected.<br>Put one space between the method name and the first argument.